File: C++/leapYear.cpp
#include<iostream>
using namespace std;

int main()
{
    int year = 2016;

    if ((year % 4 == 0) && (year % 100 != 0) || (year % 400 == 0))
    {
        cout <<"Leap year";
    }
    else
    {
        cout <<"Not leap year";
    }
    

    return 0;
}

File: Java/Problems/LeapYear.java
public class LeapYear {
    
    public static void main(String[] args) {
        
        int year = 2016;

        if ((year % 4 == 0) && (year % 100 != 0) || (year % 400 == 0)) {
            System.err.println("Leap year");
        } else {
            System.out.println("Not a leap year");
            
        }
    }
}
